CHANGELOG.md

```diff
@@ -91,7 +91,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 ### Security
 
 ## [Unreleased 2.x]
```

Comment:
Needs some cleanup ...

Reply:
Github is doing some weird stuff with the rebases. I think my branch is behind my main and then when I rebased it got confused. Sorry for all the notifications I am sure it is spamming...